src/System.IO.FileSystem.Primitives/ref/System.IO.FileSystem.Primitives.csproj

```diff
@@ -9,6 +9,9 @@
     <Compile Include="System.IO.FileSystem.Primitives.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="../../System.Runtime/ref/System.Runtime.csproj"/>
```

Comment:
Can you add a ToDo comment to remove this once packages are updated, and also log an issue for it?